assets/js/components/adminbar/AdminBarWidgets.js

```diff
@@ -81,7 +84,9 @@ export default function AdminBarWidgets() {
 					</Fragment>
 				) }
 
-				{ ( ! analyticsModuleConnected || ! analyticsModuleActive ) && (
+				{ ( ( analyticsModuleAvailable &&
+					! analyticsModuleConnected ) ||
+					! analyticsModuleActive ) && (
```

Comment:
From the previous comments, I think the intention here is:
`available && ( ! connected || ! active )`

but this is 
`( available && ! connected ) || ! active )`

Reply:
Good spot, thanks! I've fixed the condition :+1: